src/v2/view-builder/internals/BaseView.js

```diff
@@ -45,6 +45,10 @@ export default View.extend({
       return false;
     });
 
+    //make sure entire form including buttons are rendered
```

Comment:
Since we want to attach the idp buttons to `.o-form-button-bar`, we have to wait for the form button to get  rendered. I also realized we don't have an event to know if the baseView has finished rendering.

Reply:
Ah this is not good. Can't rely on timeout for form render event. Did you try triggering event from postRender() of `formController` ??

Reply:
ok, moved that bit to formController.